examples/portfolio/src/components/Button.astro

```diff
@@ -0,0 +1,13 @@
+<span>
```

Comment:
Should this actually be a `<button>`?

Reply:
We ended up not needing a component at all! (Was used twice)